modules/virt-customizing-storage-profile.adoc

```diff
@@ -58,53 +45,10 @@ metadata:
 spec:
   claimPropertySets:
   - accessModes:
-    - ReadWriteOnce <1>
+    - ReadWriteOnce
     volumeMode:
-      Filesystem <2>
-status:
+      Filesystem
+  status:
```

Comment:
`status` needs to be with the same indentation as `spec`, so we need to remove the space

Reply:
Done!